app/src/main/java/com/github/swent/echo/data/model/UserProfile.kt

```diff
@@ -5,7 +5,9 @@ data class UserProfile(
     val name: String,
     val semester: Semester?,
     val section: Section?,
-    val tags: Set<Tag>
+    val tags: Set<Tag>,
+    val committeeMember: Set<Association>,
+    val associationsSubscriptions: Set<Association>,
```

Comment:
You could add a default value, e.g. `comitteeMember: Set<Association> = emptySet()`. This way you don't need to change code all over the place. 

One other possibility is to not add this to the `UserProfile`, but as functionality to the `Repository` with functions like `getComitteMember(userId: String)`.

Reply:
After discussing this in person with @violoncelloCH and @unglazedstamp, it makes more sense to add those to the `UserProfile` instead of adding getters to the `Repository`. Also, as pointed out by @violoncelloCH a default value doesn't really make sense here. So, it's better to adapt the code everywhere...

Reply:
exactly ! thanks for writing up the conclusion here already :)